CONTRIBUTING.md

```diff
@@ -188,15 +188,22 @@ $ export GCLOUD_TESTS_GOLANG_KEYRING=projects/$GCLOUD_TESTS_GOLANG_PROJECT_ID/lo
 $ gsutil kms authorize -p $GCLOUD_TESTS_GOLANG_PROJECT_ID -k $GCLOUD_TESTS_GOLANG_KEYRING/cryptoKeys/key1
 
 # Create KMS Key in one region for Bigtable
+$ gcloud kms keyrings create $MY_KEYRING --location $MY_SINGLE_LOCATION
 $ gcloud kms keys create key1 --keyring $MY_KEYRING --location $MY_SINGLE_LOCATION --purpose encryption
 # Sets the GCLOUD_TESTS_BIGTABLE_KEYRING environment variable.
 $ export GCLOUD_TESTS_BIGTABLE_KEYRING=projects/$GCLOUD_TESTS_GOLANG_PROJECT_ID/locations/$MY_SINGLE_LOCATION/keyRings/$MY_KEYRING
+# Create a service agent, https://cloud.google.com/bigtable/docs/use-cmek#gcloud:
+gcloud beta services identity create \
+    --service=bigtableadmin.googleapis.com \
+    --project $GCLOUD_TESTS_GOLANG_PROJECT_ID
+# Note the service agent email for the agent created.
+$SERVICE_AGENT_EMAIL=<servie agent email, from last step>
```

Comment:
service is missing a c. Also I see $ at the front, this may be a mistake? I know is some shells when assigning you don't have a `$` (`SERVICE_AGENT_EMAIL=blah`)? Commands are prefixed with `$ ` in the guide,
```suggestion
$ SERVICE_AGENT_EMAIL=<service agent email, from previous step>
```

Reply:
Fixed. I think we want the $ to be consistent with the rest of the doc which shows shell prompts.  I also added an unnecessary `export` to be consistent with the rest of the doc.